### PR TITLE
#5 display snitch candidates

### PR DESCRIPTION
**Pull request by HarrisonShort**
Adds UI Canvas and Snitch prefabs, basic pressing on snitch, placeholders. Also adds TextMeshPro, so the PR is a lil heavy

sts_snitch_panel

### CODE REVIEW

File: Assets/Scripts/UIManager.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    public static UIManager Instance { get; private set; }
+
+    [SerializeField]
+    private GameObject silencePanel = null;
+
+    [SerializeField]
+    private GameObject infoPanel = null;
+
+    private List<Snitch> snitches = new List<Snitch>();
+    private Snitch selectedSnitch = null;
```
Comment (njramage) [inline Assets/Scripts/UIManager.cs]: selectedSnitch is assigned but not used. Is this intended?

**Review by njramage — APPROVED** [review]

File: Assets/Scripts/UIManager.cs (hunk 1 above)
Comment (HarrisonShort) [reply Assets/Scripts/UIManager.cs]: Yeah, this is just set up for selection later, but I realized I was probably jumping the gun on this specific ticket

**Comment by njramage** [comment]
Please rename snitch to suspect

File: Assets/Scripts/Snitch.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Snitch : MonoBehaviour
```
Comment (njramage) [inline Assets/Scripts/Snitch.cs]: Please rename class to suspect

Comment (HarrisonShort) [reply Assets/Scripts/Snitch.cs]: Have made all relevant renames (I hope)

**Pull request merged**